docs/src/app/(public)/(content)/react/overview/accessibility/page.mdx

```diff
@@ -14,7 +14,7 @@ This page highlights some of the key accessibility features of Base UI, as well
 
 Base UI components adhere to the [WAI-ARIA Authoring Practices](https://www.w3.org/WAI/ARIA/apg/) to provide basic keyboard accessibility out of the box. This is critical for users who have difficulty using a pointer device, but it's also important for users who prefer navigating with a keyboard or other input mode.
 
-Many components provide keyboard support for arrow keys, alphanumeric keys, <kbd>Home</kbd>, <kbd>End</kbd>, <kbd>Enter</kbd>, and <kbd>Esc</kbd>.
```

Comment:
Removed a word to fit it all in one line without breaking out a kbd as an orphan